api/core/throttling.py

```diff
@@ -0,0 +1,23 @@
+from django.conf import settings
+from django.core.cache import caches
+from rest_framework.request import Request
+from rest_framework.throttling import UserRateThrottle
+
+SDK_ENDPOINTS = [
+    "/api/v1/flags/",
+    "/api/v1/identities/",
+    "/api/v1/traits/",
+    "/api/v1/traits/bulk/",
+    "/api/v1/traits/increment-value/",
+    "/api/v1/environment-document/",
+]
```

Comment:
I have a feeling this is defined somewhere already? e.g. in the InfluxDBMiddleware or something? Also, for (very minor) performance benefits, should we make this a set?

Reply:
Yeah, it was defined in sentry